subsys/bluetooth/controller/ll_sw/nordic/lll/lll_adv.c

```diff
@@ -780,6 +783,32 @@ static int init_reset(void)
 	return 0;
 }
 
+#if defined(CONFIG_BT_CTLR_ZLI)
+static void mfy_pdu_free_sem_give(void *param)
+{
+	ARG_UNUSED(param);
+
+	k_sem_give(&sem_pdu_free);
+}
+
+static void pdu_free_sem_give(void)
+{
+	static memq_link_t link;
+	static struct mayfly mfy = {0, 0, &link, NULL, mfy_pdu_free_sem_give};
+	uint32_t retval;
+
+	retval = mayfly_enqueue(TICKER_USER_ID_LLL, TICKER_USER_ID_ULL_HIGH, 0,
```

Comment:
If the `k_sem_give` is delegated to be executed by myflay in `TICKER_USER_ID_ULL_HIGH` context then it is executed by `rtc0_nrf5_isr`, am I correct?

Reply:
Yes, by call to `mayfly_run(TICKER_USER_ID_ULL_HIGH);` in the `rtc0_nrf5_isr`.

https://github.com/zephyrproject-rtos/zephyr/blob/feffe639a3f8a2d2459d21a390ffd2d9d96c2437/subsys/bluetooth/controller/ll_sw/nordic/lll/lll.c#L102-L129